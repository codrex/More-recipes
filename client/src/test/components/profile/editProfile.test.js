/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import toJson from 'enzyme-to-json';
import {
  PureEditProfileForm
} from '../../../components/pages/Profile/EditProfileForm';

const props = {
  update: jest.fn(),
  handleSubmit: jest.fn(),
  submitting: false,
};

describe('EditProfileForm component ', () => {
  test('expected to match snapshot', () => {
    const component = shallow(<PureEditProfileForm {...props} />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
