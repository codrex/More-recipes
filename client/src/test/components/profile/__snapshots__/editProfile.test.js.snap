// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditProfileForm component  expected to match snapshot 1`] = `
<Form
  action=""
  className=""
  disabled={false}
  id="submit"
  lg={true}
  method=""
  onSubmit={[Function]}
  primary={false}
  primaryInverse={false}
  secondary={true}
  secondaryInverse={false}
  style={Object {}}
  submitBtnText="Update"
>
  <Field
    component={[Function]}
    id="fullname"
    name="fullname"
    placeholder="Enter fullname"
    type="text"
    validate={[Function]}
  />
  <Field
    component={[Function]}
    id="username"
    name="username"
    placeholder="Enter username"
    type="text"
    validate={[Function]}
  />
  <Field
    component={[Function]}
    id="email"
    name="email"
    placeholder="Enter email"
    type="email"
    validate={[Function]}
  />
</Form>
`;
